src/type-coercions.md

```diff
@@ -172,6 +172,8 @@ an implementation of `Unsize<U>` for `T` will be provided:
 
 * `T` to `dyn U`, when `T` implements `U + Sized`, and `U` is [object safe].
 
+* `dyn T` to `dyn U`, when `U` is one of `T`'s supertraits.
```

Comment:
This should also mention that upcasting coercions allow dropping auto traits. i.e. `dyn Trait + Send -> dyn Trait`.